tests/python/pants_test/backend/jvm/tasks/test_jvm_run_integration.py

```diff
@@ -12,13 +12,7 @@ def _exec_run(self, target, *args):
         :param args: list of arguments to append to the command
         :return: stdout as a string on success, raises an Exception on error
         """
-        # Avoid some known-to-choke-on interpreters.
-        command = [
-            "run",
-            target,
-            "--python-setup-interpreter-constraints=CPython>=2.7,<3",
-            "--python-setup-interpreter-constraints=CPython>=3.3",
-        ] + list(args)
+        command = ["run", target] + list(args)
```

Comment:
Nit: we've recently been using the idiom `["run", target, *args]`. This is subjective, but the argument is for readability and being able to avoid the call to `list()`.

Reply:
Fixed.